jumanji/training/setup_train.py

```diff
@@ -61,6 +62,9 @@
 
 def setup_logger(cfg: DictConfig) -> Logger:
     logger: Logger
+    # Log only once if there are multiple hosts on the pod.
```

Comment:
To be clear, _host_ and _worker_ are referring to the same thing, right?

Reply:
I guess host is more general than worker? But yes.

Reply:
Ok, maybe we should stick with "worker" then? Since we use that in variable names

Reply:
The logger is different from multi-worker training, isn't it? I will stick to this to go fast since they are the same thing.